funnel/transports/base.py

```diff
@@ -43,4 +45,16 @@ def init():
     ):
         platform_transports['sms'] = True
 
+    # We need to set Callback URL for SMS notifications here as there are two different providers
+    # (Exotel and Twilio) and both of them need different URLs as the schema is very different.
+    with app.app_context():
+        app.config['SMS_TWILIO_CALLBACK'] = url_for(
+            'process_twilio_event', _external=True, _method="POST"
```

Comment:
Our string quote convention is single quotes for identifiers and double quotes for human-facing text. In this case, it should be `'POST'` in single quotes.